openmetadata-ui/src/main/resources/ui/src/components/Database/DatabaseSchema/DatabaseSchemaTable/DatabaseSchemaTable.tsx

```diff
@@ -160,6 +184,95 @@ export const DatabaseSchemaTable = ({
     }
   };
 
+  const handleDisplayNameUpdate = useCallback(
+    async (data: EntityName, id?: string) => {
+      try {
+        const schemaDetails = schemas.find((schema) => schema.id === id);
+        if (!schemaDetails) {
+          throw new Error(t('error.tableNotFound'));
```

Comment:
Can you re-check the translation file, I don't think we don't have any error JSON in the translation file.

Reply:
Corrected it.